Missionframework/scripts/client/ammoboxes/ammobox_action_manager.sqf

```diff
@@ -57,10 +57,12 @@ while {true} do {
                 _b_action_id2 = _next_box addAction ["<t color='#FFFF00'>" + localize "STR_ACTION_STORE_CRATE" + "</t>",{[(_this select 0), (nearestObjects [player,KPLIB_storageBuildings,20]) select 0,true] call KPLIB_fnc_crateToStorage;},"",-502,true,true,"","build_confirmed == 0 && (_this distance _target < 5) && (vehicle player == player)"];
                 _b_action_id3 = _next_box addAction ["<t color='#FFFF00'>" + localize "STR_ACTION_CRATE_VALUE" + "</t>",{[_this select 0] call KPLIB_fnc_checkCrateValue;uiSleep 3; hint "";},"",-503,true,true,"","build_confirmed == 0 && (_this distance _target < 5) && (vehicle player == player)"];
                 _b_action_id4 = _next_box addAction ["<t color='#FFFF00'>" + localize "STR_ACTION_CRATE_PUSH" + "</t>",{(_this select 0) setPos ((_this select 0) getPos [1, (player getDir (_this select 0))]);},"",-504,true,false,"","build_confirmed == 0 && (_this distance _target < 5) && (vehicle player == player)"];
+                _b_action_id5 = _next_box addAction ["<t color='#FFFF00'>" + localize "STR_ACTION_CRATE_CARRY" + "</t>",{(_this select 0) attachTo [(_this select 1), [0, 2, 1]];},"",-504,true,false,"","build_confirmed == 0 && (_this distance _target < 5) && (vehicle player == player) && (isNull attachedTo _target)"];
```

Comment:
Having this added here is "fine enough" atm. Basically the whole ammobox_action_manager should be removed in the upcoming changes.
Until then it's fine. Wouldn't make sense to refactor the ammobox action manager in the scope of the carry action.